routes/users.js

```diff
@@ -208,6 +208,8 @@ router.get('/isUsernameAvailable/:username', authenticate, users.getUsernameAvai
  *             schema:
  *               $ref: '#/components/schemas/errors/badImplementation'
  */
+router.get('/chaincode', authenticate, users.generateChaincode)
```

Comment:
Please change the position of this route as the above swagger documentation is for `users/:username` route